app.rb

```diff
@@ -74,8 +74,8 @@
     win: "3.8.1"
   },
   :v4 => {
-    linux: "4.0.1",
-    win: "4.0.1"
+    linux: "4.1.0",
+    win: "4.1.0"
```

Comment:
td-agent 4.1.0 for Windows isn't release yet.

https://github.com/fluent-plugins-nursery/td-agent-builder/pull/269#issuecomment-784921333
https://td-agent-package-browser.herokuapp.com/4/windows